src/routes/controllers/users.controller.js

```diff
@@ -117,3 +121,26 @@ export async function getReadingStats(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+/**
+ *
+ * @param {Object} req express request
+ * @param {Object} res express response
+ * @returns {Array} user likes
+ */
+export async function getAllArticlesLikedByUser(req, res) {
+  const { user } = req;
+
+  try {
+    const currentUser = await getUserbyId(user.id);
+    const allArticlesLikedByUser = await currentUser.getLikes();
+    return successResponse(
+      res,
+      200,
+      'Successfully retrieved all articles liked by user',
```

Comment:
Could we make this response statement a bit shorter? It looks wordy
Maybe something along the lines of 'Successfully retrieved all liked articles'

Reply:
This response message would not be specific to the data that the response is displaying. This suggested response message refers to all articles that have a like whether by the current logged in user or not. We are returning all articles that the user has liked on the platform

Reply:
We can remove this success response message.

Reply:
This has been resolved in another PR